x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable_factory.ts

```diff
@@ -7,7 +7,7 @@
 import { Capabilities, HttpSetup } from 'kibana/public';
 import { i18n } from '@kbn/i18n';
 import { RecursiveReadonly } from '@kbn/utility-types';
-import { toExpression, Ast } from '@kbn/interpreter/target/common';
+import { Ast } from '@kbn/interpreter/target/common';
```

Comment:
Don't load code from interpreter within the factory (because it's part of the initial bundle)